src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,15 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  category: [],
+};
+
+const categorySlice = createSlice({
+  name: 'category',
+  initialState,
+  reducers: {
+    checkStatus: () => 'Under construction',
+  },
+});
+export const { checkStatus } = categorySlice.actions;
+export default categorySlice.reducer;
```

Comment:
- [ ] Please see below how your CategorySlice file code show be 👇🏾. Kindly implement the `checkStatus` reducer as shown below.

```javascript
// ./src/redux/categories/categoriesSlice.js

import { createSlice } from "@reduxjs/toolkit"

export const categoriesSlice = createSlice({
  name: "categories",
  initialState: {
    categories: [],
  },
  reducers: {
    checkStatus: (state, action) => {
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories
    },
  },
})

export const { checkStatus } = categoriesSlice.actions

export default categoriesSlice.reducer
```

Reply:
I got it. Thank you.